### ISSUE TITLE
Changing the terminal size did not update the progress bar length

### ISSUE THREAD

**Issue by mschaaf** [willbenica/po_tools#13]
1. run the script with a progress bar
2. make the terminal smaller so that the progress bar goes on the next line
3. always see a scrolling

Expected behaviour

Adopt the progress bar to the screen size to avoid scrolling.

**Comment by willbenica** [comment]
@mschaaf,
I've added the following bit of code to the script:
```
generated = tqdm(generate_modified_data(template_data, iterations, num_folders, dm_folder),
                         total=iterations,
                         dynamic_ncols=True
                         )
```
the `dynamic_ncols=True` parameter should solve this issue. But it doesn't exactly :-(

When making the terminal larger, this does what is expected - increases the length of the length of the progress bar. But when making a terminal window smaller, I still get the line breaks with duplicate progress bars.

From what I could see at the [tqdm github page](https://github.com/tqdm/tqdm/issues/203), this seems to be a known issue.

I've added the `dynamic_ncols` parameter to branch `enhancement/parallel_imports` if you want to give it a try in your terminal.
